docs/graphql/manual/deployment/heroku/updating.rst

```diff
@@ -50,6 +50,8 @@ When you ``git push`` to deploy, the Heroku app will get updated with the latest
 
    $ git push heroku master
 
+If you are getting the following error: ``fatal: 'heroku' does not appear to be a git repository``, add your heroku repo to your local git remote via: ``git remote add heroku https://git.heroku.com/<you-app-name>.git``. (this URL locates in your Heroku - Settings - Info - Heroku Git URL)
```

Comment:
Actually, this part is already covered in the above section "Step 2: Attach your Heroku app". If the above step succeeds, there won't be an error after `git push heroku master`. But you could add the part in brackets to step 2, so that the user knows how to locate the Heroku repo.

Reply:
Ah, makes sense, thanks! done. 
No idea why I still got that error back to that time, must do some wrong. :D

Reply:
One last thing: what do you mean by " Go to the repo folder you just downloaded...". I don't think anything is downloaded in this step. How about: "Navigate to your project directory, use the..."? 🙂

Reply:
Haha, for that part, this is my first confusion, the description of this section seems to just clone the repo, and run the commands in section 2. 

But after cloning the repo, you are actually outside the repo, since I always use git cli, and I am not sure about this how does heroku wrapper `heroku git:remote` actually works, so I just wonder since the documentation didn't say `cd repo-folder`, maybe the heroku wrapper will automatically `cd` that folder for me? haha

After I figuring out there is no magic, it is already cost me some time, so I just want to emphasize this part to ease the pain for the others, most dev uses git, but not everyone uses heroku before. :D

Reply:
Sure, this I understand. It should be clear that you have to move inside the folder. But the `download` thing is confusing I think. So can we change it to: "Navigate to your project directory, use the..."?

Reply:
makes a lot of sense to me, done! :D

Reply:
Thank you so much for contributing to our docs 😊